### PR TITLE
net-mgmt/net-snmp Add Observium extended OIDs

### PR DESCRIPTION
**Pull request by pmhausen**
Add OIDs and script for the Observium network management system.

https://docs.observium.org/device_linux/

(Adapted for FreeBSD)

### CODE REVIEW

File: net-mgmt/net-snmp/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		net-snmp
-PLUGIN_VERSION=		1.5
+PLUGIN_VERSION=		1.6
 PLUGIN_REVISION=	5
```
Comment (fichtner) [inline net-mgmt/net-snmp/Makefile:3]:
```suggestion
PLUGIN_VERSION=		1.6
```

Comment (pmhausen) [reply net-mgmt/net-snmp/Makefile]: Done.

File: net-mgmt/net-snmp/src/sbin/distro [hunk 2]
```diff
@@ -0,0 +1,605 @@
+#!/bin/sh
```
Comment (fichtner) [inline net-mgmt/net-snmp/src/sbin/distro:1]: I don't want to vet this whole script (especially the license part of it and how it interacts with the bsd license of the plugin in general), but I would suggest moving it to `src/opnsense/scripts/OPNsense/Netsnmp/distro.sh` just to get it out of the standard system path and allow additional syntax checking on it (and I like the fact that it's a POSIX script)

Comment (pmhausen) [reply net-mgmt/net-snmp/src/sbin/distro:1]: Where will this end up in the installed plugin, please? So I can also adapt the path in the snmpd configuration.

Thanks.

Comment (fichtner) [reply net-mgmt/net-snmp/src/sbin/distro:1]: src = /usr/local

Comment (pmhausen) [reply net-mgmt/net-snmp/src/sbin/distro]: Done.

File: net-mgmt/net-snmp/src/opnsense/scripts/OPNsense/Netsnmp/setup.sh [hunk 3]
```diff
@@ -7,3 +7,5 @@ chmod 755 /var/net-snmp
 mkdir -p /var/net-snmp/mib_indexes
 chown -R root:wheel /var/net-snmp/mib_indexes
 chmod 700 /var/net-snmp/mib_indexes
+
+chmod 755 /usr/local/sbin/distro
```
Comment (fichtner) [inline net-mgmt/net-snmp/src/opnsense/scripts/OPNsense/Netsnmp/setup.sh:11]:
```suggestion
```

let me fix that in git after merge which is a bot more elegant and required for scripts content anyway

File: net-mgmt/net-snmp/src/opnsense/mvc/app/models/OPNsense/Netsnmp/General.xml [hunk 4]
```diff
@@ -31,6 +31,10 @@
             <default>0</default>
             <Required>Y</Required>
         </enableagentx>
+        <enableobservium type="BooleanField">
```
Comment (fichtner) [inline net-mgmt/net-snmp/src/opnsense/mvc/app/models/OPNsense/Netsnmp/General.xml:34]: same here, model bump for new default to render

File: net-mgmt/net-snmp/src/opnsense/scripts/OPNsense/Netsnmp/setup.sh (hunk 3 above)
Comment (pmhausen) [reply net-mgmt/net-snmp/src/opnsense/scripts/OPNsense/Netsnmp/setup.sh:11]: I already fixed it in git. Sorry, forgot to remove that.

**Comment by fichtner** [comment]
Merged, thanks!

File: net-mgmt/net-snmp/src/opnsense/mvc/app/models/OPNsense/Netsnmp/General.xml (hunk 4 above)
Comment (pmhausen) [reply net-mgmt/net-snmp/src/opnsense/mvc/app/models/OPNsense/Netsnmp/General.xml:34]: Done.

**Pull request merged**